src/main/java/skytils/skytilsmod/features/impl/dungeons/DungeonsFeatures.java

```diff
@@ -38,12 +37,28 @@
 public class DungeonsFeatures {
 
     private static final Minecraft mc = Minecraft.getMinecraft();
-    
+
     @SubscribeEvent(priority = EventPriority.HIGHEST)
     public void onChat(ClientChatReceivedEvent event) {
         if (!Utils.inSkyblock) return;
         String unformatted = StringUtils.stripControlCodes(event.message.getUnformattedText());
 
+
+        if (Skytils.config.hideAbilities && Utils.inDungeons) {
+            if (unformatted.contains("is now available!") || unformatted.contains("is ready to use!")) {
+                event.setCanceled(true);
+            }
+        }
+        if (Skytils.config.hideBlessings && Utils.inDungeons) {
+            if (unformatted.startsWith("DUNGEON BUFF!") || unformatted.startsWith("A Blessing") || unformatted.contains("has obtained Blessing of") || unformatted.contains("Grants you") || unformatted.contains("Granted you") || unformatted.contains("found a Wither Essence!")) {
+                event.setCanceled(true);
+            }
+        }
+        if (Skytils.config.hideMilestones && Utils.inDungeons) {
+            if (unformatted.startsWith("Mage Milestone") || unformatted.startsWith("Berserk Milestone") || unformatted.startsWith("Archer Milestone") || unformatted.startsWith("Tank Milestone") || unformatted.startsWith("Healer Milestone")) {
```

Comment:
Why not change this to just a `.contains("Milestone")`

Reply:
What if someone in the party says "I need Milestones"
If you went this route you would need regex